From 3ac3253fea164837c527f56f9dd923ac559b11ef Mon Sep 17 00:00:00 2001
From: Yangbo Lu <yangbo.lu@nxp.com>
Date: Wed, 5 Dec 2018 16:31:13 +0800
Subject: [PATCH 726/767] sdk_dpaa: remove useless 1588 timer enablement

1588 timer had been enabled to run at ptp_qoriq driver
probing stage. So removed the useless enablement.

Signed-off-by: Yangbo Lu <yangbo.lu@nxp.com>
[Xulin: Original patch taken from NXP LSDK-18.12.]
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 drivers/net/ethernet/freescale/sdk_dpaa/dpaa_eth_common.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/drivers/net/ethernet/freescale/sdk_dpaa/dpaa_eth_common.c b/drivers/net/ethernet/freescale/sdk_dpaa/dpaa_eth_common.c
index 1d9d723575d8..2723f137941e 100644
--- a/drivers/net/ethernet/freescale/sdk_dpaa/dpaa_eth_common.c
+++ b/drivers/net/ethernet/freescale/sdk_dpaa/dpaa_eth_common.c
@@ -340,8 +340,6 @@ static void dpa_ts_tx_enable(struct net_device *dev)
 	struct dpa_priv_s *priv = netdev_priv(dev);
 	struct mac_device *mac_dev = priv->mac_dev;
 
-	if (mac_dev->fm_rtc_enable)
-		mac_dev->fm_rtc_enable(get_fm_handle(dev));
 	if (mac_dev->ptp_enable)
 		mac_dev->ptp_enable(mac_dev->get_mac_handle(mac_dev));
 
@@ -373,8 +371,6 @@ static void dpa_ts_rx_enable(struct net_device *dev)
 	struct dpa_priv_s *priv = netdev_priv(dev);
 	struct mac_device *mac_dev = priv->mac_dev;
 
-	if (mac_dev->fm_rtc_enable)
-		mac_dev->fm_rtc_enable(get_fm_handle(dev));
 	if (mac_dev->ptp_enable)
 		mac_dev->ptp_enable(mac_dev->get_mac_handle(mac_dev));
 
-- 
2.17.0

